### ISSUE TITLE
Server Maintenance Mode

### ISSUE THREAD

**Issue by luke6464** [nextcloud/android#12002]
### ⚠️ Before posting ⚠️

- [X] This is a **bug**, not a question or an enhancement.
- [X] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [X] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

Nextcloudpi Server (1.51.6) on Raspberry with Nextcloud 25.0.9

### Expected behaviour

Normal access to the server.

### Actual behaviour

For a few days now, with the latest update of the app, after having successfully connected to the server it reports that the server is in maintenance mode and does not display the files and folders...
however the server works fine from the browser and is not in maintenance mode! ...
Attached are the screenshots of the app and browser!
So is this a problem with the app? ...
Maybe you introduced a bug in the latest app update? ...
how do I solve the problem? ...
I have already tried uninstalling and reinstalling the app several times without being able to resolve the problem...
on the server I am using NextCloud version 25.0.9.
Thank you
App NextCloud
![Screenshot_20231004-100925_Nextcloud](https://github.com/nextcloud/android/assets/3793119/2692f7ee-9cb6-42ad-8b16-875b2c312871)
Browser
![screenshot_20231004_101228](https://github.com/nextcloud/android/assets/3793119/55f4ed56-ee28-4235-b822-8b302f32e457)

### Android version

9

### Device brand and model

Samsung Note 8

### Stock or custom OS?

Stock

### Nextcloud android app version

3.26

### Nextcloud server version

Nextcloudpi Server (1.51.6) on Raspberry with Nextcloud 25.0.9

### Using a reverse proxy?

Yes

### Server error logs

```bash
2023-10-02 10:16:37 0 [Note] InnoDB: Setting file './ibtmp1' size to 12 MB. Physically writing the file full; Please wait ...
2023-10-02 10:16:37 0 [Note] InnoDB: File './ibtmp1' size is now 12 MB.
2023-10-02 10:16:37 0 [Note] InnoDB: 10.5.19 started; log sequence number 7211761387; transaction id 24800753
2023-10-02 10:16:37 0 [Note] Plugin 'FEEDBACK' is disabled.
2023-10-02 10:16:37 0 [Note] InnoDB: Loading buffer pool(s) from /data/database/ib_buffer_pool
2023-10-02 10:16:37 0 [Note] Server socket created on IP: '127.0.0.1'.
2023-10-02 10:16:39 0 [Note] Reading of all Master_info entries succeeded
2023-10-02 10:16:39 0 [Note] Added new Master_info '' to hash table
2023-10-02 10:16:39 0 [Note] mysqld: ready for connections.
Version: '10.5.19-MariaDB-0+deb11u2'  socket: '/run/mysqld/mysqld.sock'  port: 3306  Debian 11
Starting Redis
Starting Cron
Starting Postfix
2023-10-02 10:16:41 0 [Note] InnoDB: Buffer pool(s) load completed at 231002 10:16:41
      
Provisioning
Provisioning Redis password
Updating NextCloud config with Redis password
Provisioning MariaDB password
Updating NextCloud config with MariaDB password
Running nc-limits
[ nc-limits ] (Mon Oct  2 10:16:47 UTC 2023)
Using 6143855616 for PHP max memory
Installing template 'php/90-ncp.ini.sh'...
Using 6143855616 for PHP max memory
Installing template 'php/90-ncp.ini.sh'...
Using 78 PHP threads
Installing template 'php/pool.d.www.conf.sh'...
Using 78 PHP threads
Installing template 'php/pool.d.www.conf.sh'...
Installing template 'mysql/91-ncp.cnf.sh'...
Installing template 'mysql/91-ncp.cnf.sh'...
Stopping redis-server: redis-server.
Stopping redis-server: redis-server.
Starting redis-server: redis-server.
Starting redis-server: redis-server.
Starting notify_push daemon
Configuring Domain
System config value trusted_domains => 11 set to string 185.8.198.101
System config value trusted_domains => 1 set to string 172.17.0.2
System config value trusted_domains => 14 set to string nextcloud
System config value trusted_domains => 3 set to string nextcloud
System config value overwrite.cli.url set to string https://nextcloud/
System config value trusted_proxies => 11 set to string 127.0.0.1
System config value trusted_proxies => 12 set to string ::1
System config value trusted_proxies => 13 set to string nextcloud
System config value trusted_proxies => 14 set to empty string
Setup notify_push (attempt 1/5)
✓ redis is configured
✓ push server is receiving redis messages
✓ push server can load mount info from database
✓ push server can connect to the Nextcloud server
✓ push server is a trusted proxy
✓ push server is running the same version as the app
  configuration saved
Nextcloud version 25.0.9.2. NextCloudPi version v1.51.6
Back up current instance...
check free space...
clean up old backup /data/docker-startup-backups/nextcloud-bkp_20231001_1696120833.tar.gz
Maintenance mode enabled
backup database...
backup files...
backup /data/docker-startup-backups/nextcloud-bkp_20231002_1696241848.tar.gz generated
Maintenance mode disabled
      
e
[ nc-previews-auto ]
[ nc-previews-auto ]
2023-10-04  7:58:29 2954 [Warning] Aborted connection 2954 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  7:58:30 2955 [Warning] Aborted connection 2955 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  7:58:31 2961 [Warning] Aborted connection 2961 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  7:58:31 2963 [Warning] Aborted connection 2963 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  7:58:33 2967 [Warning] Aborted connection 2967 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  7:58:33 2966 [Warning] Aborted connection 2966 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  7:58:34 2970 [Warning] Aborted connection 2970 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  7:58:34 2971 [Warning] Aborted connection 2971 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  7:58:52 2974 [Warning] Aborted connection 2974 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  8:09:14 3072 [Warning] Aborted connection 3072 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  8:09:15 3073 [Warning] Aborted connection 3073 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  8:09:16 3079 [Warning] Aborted connection 3079 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  8:09:16 3081 [Warning] Aborted connection 3081 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  8:09:18 3084 [Warning] Aborted connection 3084 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  8:09:18 3085 [Warning] Aborted connection 3085 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  8:09:18 3088 [Warning] Aborted connection 3088 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  8:09:19 3089 [Warning] Aborted connection 3089 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  9:15:29 3339 [Warning] Aborted connection 3339 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  9:15:30 3344 [Warning] Aborted connection 3344 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  9:15:30 3345 [Warning] Aborted connection 3345 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  9:15:31 3349 [Warning] Aborted connection 3349 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  9:15:31 3350 [Warning] Aborted connection 3350 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  9:16:46 3352 [Warning] Aborted connection 3352 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  9:16:46 3355 [Warning] Aborted connection 3355 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  9:16:47 3357 [Warning] Aborted connection 3357 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  9:18:31 3360 [Warning] Aborted connection 3360 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  9:18:32 3363 [Warning] Aborted connection 3363 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  9:18:33 3365 [Warning] Aborted connection 3365 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  9:33:06 3387 [Warning] Aborted connection 3387 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  9:33:07 3390 [Warning] Aborted connection 3390 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
2023-10-04  9:33:07 3392 [Warning] Aborted connection 3392 to db: 'nextcloud' user: 'ncadmin' host: 'localhost' (Got an error reading communication packets)
```

**Comment by joshtrichards** [comment]
Can you provide a few of the transactions that appear in your `nextcloud.log` when you either first open the Android client or first pull down on the client's screen to refresh it?

**Comment by joshtrichards** [comment]
I wonder if this is at all related to: https://github.com/nextcloud/android/issues/11982#issuecomment-1730500510

**Comment by luke6464** [comment]
`Also, just to rule it the obvious: if you select your profile image in the upper right is there only one account configured?`
Yess

`Can you provide a few of the transactions that appear in your nextcloud.log when you either first open the Android client or first pull down on the client's screen to refresh it?`
Where is file nextcloud.log? ... how do I find it on the cellular (Samsung) Note 8 - Android 9?

I can confirm that the BUG was introduced with the latest version... in fact I installed the previous version of the APP (3.25.0) and the problem does not occur.
The problem only affects the latest version 3.26.0!!

Thanks

**Comment by joshtrichards** [comment]
> I can confirm that the BUG was introduced with the latest version... in fact I installed the previous version of the APP (3.25.0) and the problem does not occur. The problem only affects the latest version 3.26.0!!

Okay, in that case at your next convenience let us know if it is fixed in the recently v3.27.0 for you. Other reports of similar behavior show this being fixed in v3.27: https://github.com/nextcloud/android/issues/11982#issuecomment-1864446714
